utils/view-engine.js

```diff
@@ -1,19 +1,14 @@
 async function engine(filePath, options, callback) {
   try {
-    if (!options.cache) {
-      delete require.cache[require.resolve(filePath)];
-    }
```

Comment:
This allowed us to not require a server restart in dev mode when a view has changed. ESM has no concept of cache clearing yet.

Reply:
Note to self: It looks like ESM import allows URL params, which can be used to re-import as if cache is not present. It'll lead to memory leak as there is no way to unload (maybe except GC), but it might be fine in dev env, and allow us to reload a view without server restart.